pages/EC-WebSphere_help.xml

```diff
@@ -3435,8 +3435,11 @@
         <h1>Release Notes</h1>
         <h2>@PLUGIN_KEY@ 2.2.1</h2>
         <ul>
-             <li>Update application in DeployEnterpriseApp procedure if it is already deployed.</li>
-             <li>Added StartCluster, StopCluster, MapSharedLibrary, modifyApplicationClassLoader procedures.</li>
+            <li>DeployEnterpriseApp procedure was refactored.</li>
+            <li>DeployEnterpriseApp procedure from now updates application, if it is already deployed.</li>
```

Comment:
DeployEnterpriseApp procedure will now update the application if it is already deployed.